synapseclient/table.py

```diff
@@ -572,21 +584,25 @@ class EntityViewSchema(SchemaBase):
     """
     A EntityViewSchema is a :py:class:`synapseclient.entity.Entity` that displays all files/projects (depending on user choice) within a given set of scopes
```

Comment:
For EntityViewSchema's documentation could you also rename `view_type` ->`type`? This documentation error fix is for [ SYNPY-629](https://sagebionetworks.jira.com/browse/SYNPY-629)

Reply:
@zimingd that parameter had already been renamed in the documentation